core/contracts/financial-templates/implementation/PricelessPositionManager.sol

```diff
@@ -213,6 +214,10 @@ contract PricelessPositionManager is FeePayer, AdministrateeInterface {
     {
         PositionData storage positionData = _getPositionData(msg.sender);
         require(positionData.requestPassTimestamp == 0);
+        require(
+            collateralAmount.isGreaterThan(0) &&
+                collateralAmount.isLessThanOrEqual(_getCollateral(positionData.rawCollateral))
```

Comment:
This check feels redundant as we would have a safe math underflow from the call to `_removeCollateral` if this inequality was not true, would we not? I'm all for expressiveness but it might be redundant.

Reply:
Yeah I think that's right. I still think we should keep the `isGreaterThan(0)` part wdyt?